### ISSUE TITLE
Pearson's Chi-Sq test comparing predictor fn and histogram

### ISSUE THREAD

**Issue by schanzer** [brownplt/pyret-lang#1380]
Advanced DS request from Steve Ornburn, a teacher in Houston:
![image](https://user-images.githubusercontent.com/77126/43203649-ceecc31a-8fec-11e8-9d90-6e34edb9a5ed.png)

"I’d like to be able to superimpose the [graph of a function] over a histogram of the data used to estimate the parameters.  The stretch request, I’d like to do a Pearson’s chi-square test to evaluate the it between the histogram and the PDF. "

**Comment by sorawee** [comment]
Yeah, Google Charts currently don't have this ability. Bar charts, line charts, and scatter plot can be super imposed together, but histogram is not in the family.

(See https://developers.google.com/chart/interactive/docs/gallery/combochart)
